package/MDAnalysis/analysis/encore/confdistmatrix.py

```diff
@@ -67,18 +68,26 @@ def conformational_distance_matrix(ensemble,
     conf_dist_function : function object
         Function that fills the matrix with conformational distance
         values. See set_rmsd_matrix_elements for an example.
-    pairwise_align : bool
+    selection : str, optional
+        use this selection
+    superimposition_selection : str, optional
+        TODO
```

Comment:
@mtiberti @wouterboomsma do you have a good idea for the doc description of this parameter. There isn't one in develop right now